### PR TITLE
Refactor SignTransactionWeb3 to not use scss anymore

### PR DESCRIPTION
**Pull request by blurpesec**

### CODE REVIEW

File: src/components/SignTransactionWallets/Web3.tsx [hunk 1]
```diff
@@ -107,61 +115,110 @@ export default function SignTransactionWeb3({
...
+        {walletState === WalletSigningState.ADDRESS_MISMATCH && (
+          <InlineMessage textAlign="center">
+            {translate('SIGN_TX_WEB3_FAILED_ACCOUNT', {
+              $walletName: walletConfig.name,
+              $address: senderAccount.address
+            })}
+          </InlineMessage>
+        )}
+        {walletState === WalletSigningState.SUBMITTING && (
+          <InlineMessage textAlign="center" type={InlineMessageType.INFO_CIRCLE}>
```
Comment (FrederikBolding) [inline src/components/SignTransactionWallets/Web3.tsx:205]: I think we should use another color for this notice, it looks like a warning.
image

File: src/components/SignTransactionWallets/Web3.tsx [hunk 2]
```diff
@@ -107,61 +115,110 @@ export default function SignTransactionWeb3({
...
+  senderAccount: StoreAccount;
+}
+
+export const SignTransactionWeb3UI = ({
+  walletConfig,
+  walletState,
+  networkName,
+  senderAccount
+}: UIProps) => (
+  <Box p="2.5em">
```
Comment (FrederikBolding) [inline src/components/SignTransactionWallets/Web3.tsx:166]: I don't think we need padding at the bottom as there is already padding from an outer box.

Comment (FrederikBolding) [reply src/components/SignTransactionWallets/Web3.tsx:166]: Also for the sides, it looks squished compared to `master`.

File: src/components/SignTransactionWallets/Web3.tsx [hunk 3]
```diff
@@ -107,61 +115,110 @@ export default function SignTransactionWeb3({
...
+            })}
+          </InlineMessage>
+        )}
+        {walletState === WalletSigningState.SUBMITTING && (
+          <InlineMessage textAlign="center" type={InlineMessageType.INFO_CIRCLE}>
+            {translate('SIGN_TX_SUBMITTING_PENDING')}
+          </InlineMessage>
+        )}
+      </Box>
+      <Body textAlign="center" fontSize="2" paddingTop="16px">
```
Comment (FrederikBolding) [inline src/components/SignTransactionWallets/Web3.tsx:210]: Prefer margin

File: src/components/SignTransactionWallets/Web3.tsx [hunk 4]
```diff
@@ -107,61 +115,110 @@ export default function SignTransactionWeb3({
...
   );
 }
+
+const Footer = styled.div`
+  width: 100%;
+  margin-top: 2em;
+`;
+
+const Web3ImgContainer = styled.div`
+  padding-bottom: 3em;
```
Comment (FrederikBolding) [inline src/components/SignTransactionWallets/Web3.tsx:133]: Add the margin that we had before on all sides

File: src/components/InlineMessage/InlineMessage.tsx [hunk 5]
```diff
@@ -83,11 +84,12 @@ export default function InlineMessage({
...
   value,
   children,
-  className
+  className,
+  textAlign
 }: Props) {
   const config = messageConfig(type);
   return (
-    <Wrapper className={className}>
+    <Wrapper className={className} textAlign={textAlign}>
```
Comment (FrederikBolding) [inline src/components/InlineMessage/InlineMessage.tsx:92]: Is there a reason not to create a styled component for this in `Web3.tsx`?

**Review by FrederikBolding — CHANGES_REQUESTED** [review]
Small styling tweaks

File: src/components/SignTransactionWallets/Web3.tsx (hunk 1 above)
Comment (blurpesec) [reply src/components/SignTransactionWallets/Web3.tsx:205]: We use this color for inline messaging in Send Assets too. Would need some feedback from design to change that
image

Comment (FrederikBolding) [reply src/components/SignTransactionWallets/Web3.tsx:205]: Yes. For warnings not for info

Comment (blurpesec) [reply src/components/SignTransactionWallets/Web3.tsx:205]: image
dis betta color?

Comment (FrederikBolding) [reply src/components/SignTransactionWallets/Web3.tsx]: I think it is fine to use white unless we have a design decision to use another color.

File: src/components/InlineMessage/InlineMessage.tsx [hunk 6]
```diff
@@ -25,7 +26,7 @@ interface Config {
 export const Wrapper = styled.div`
   font-size: ${FONT_SIZE.BASE};
   width: 100%;
-  text-align: justify;
+  text-align: 'justify';
```
Comment (FrederikBolding) [inline src/components/InlineMessage/InlineMessage.tsx:29]: Is the necessary?

File: src/components/SignTransactionWallets/Web3.tsx (hunk 2 above)
Comment (FrederikBolding) [reply src/components/SignTransactionWallets/Web3.tsx]: This has not been resolved. I think the padding can be removed entirely as the outer `section` has the needed padding already.

File: src/components/SignTransactionWallets/Web3.tsx (hunk 1 above)
Comment (blurpesec) [reply src/components/SignTransactionWallets/Web3.tsx]: Changed to purple as per taylor

**Review by FrederikBolding — APPROVED** [review]

**Pull request merged**